nss-day-cohort-48/giffygram-altavista-giffygram
# added message provider

#### Changes Made

1. Renamed `src/scripts/message` to `src/scripts/Message`.
2. Added file `MessageProvider.js` to `src/scripts/Message` directory.
3. Modified file `provider.js` to include `fetchMessages()` and `rawMessages()`.
4. Renamed `src/scripts/data/provider.js` to `src/scripts/data/dataAccess.js`

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout branchname
   ```
2. Open a new Terminal and open the giffygram repo (make sure it's named 'giffygram')
3. Run `./serve-giffygram` to serve the code.
4. Test app functionality.
   > Check the console to see the new message layout
   > Everything else should be unchanged
5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
